src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -113,6 +135,8 @@ private void DisableWindowsUpdate()
                         auUtility.SetAUOptions();
                         _eventSource.InfoMessage("New AU registry values are {0}", auUtility.LogCurrentAUValues());
                     }
+                    string updateMsg = "Automatic Windows update disabled successfully.";
+                    this._nodeAgentSfUtility.ReportHealth("WindowsUpdateOperationResult", updateMsg, HealthState.Ok, -1, TimeSpan.FromMinutes(this._serviceSettings.OperationTimeOutInMinutes));
```

Comment:
>WindowsUpdateOperationResult [](start = 59, length = 28)

1. Define the property in the class. 
2. Change the name to WindowsUpdateSetting